proto/pylons/pylons/tx.proto

```diff
@@ -242,4 +243,15 @@ message MsgUpdateCookbook {
 }
 
 message MsgUpdateCookbookResponse {
+}
+
+message MsgRegisterKYCAddress {
+  string creator = 1;
+  string username = 2;
+  uint64 level = 3;
+  string provider = 4;
+  string provider_id = 5;
+}
+
```

Comment:
add MsgUnregisterKYCAddress

Reply:
@MikeSofaer 
I understand that we may need to unregister KYC address in the future in case of violation or some sort. However, unregistering an address while it still has ubedrock in it may require forcefully taking ubedrock out of that address and move it to a founding team wallet.

What is your thoughts on this?